docs/design/host_to_services_OpenFlow.md

```diff
@@ -46,6 +46,43 @@ OVN masquerade IPs are 169.254.169.1, and fd69::1
 
 Another worker node exists, ovn-worker2 at 172.18.0.4.
 
+### OVN masquerade addresses
+- IPv4
+  - 169.254.169.1: OVN masquerade address
+  - 169.254.169.2: host masquerade address
+  - 169.254.169.3: local ETP masquerade address (not used for shared GW, kept for parity)
+  - 169.254.169.4: dummy next-hop masquerade address
+- IPv6
+  - fd69::1: OVN masquerade address
+  - fd69::2: host masquerade address
+  - fd69::3: local ETP masquerade address (not used for shared GW, kept for parity)
+  - fd69::4: dummy next-hop masquerade address
+
+### Nodes without a default GW
+For scenarios where the nodes do not have a known next-hop (microshift, for instance), we rely on a dummy next-hop
+masquerade IP address.
+
+We need to provision this (fake) neighbor in OVN, enabling ARP / ND replies for this fake masquerade IP address.
+
+OVN is also provisioned with a static route specifying this dummy next-hop masquerade IP as its default route.
+
+Finally, to enable the hairpin scenario, we need to provision a route specifying traffic originating from the host
+must be routed to the OVN masquerade address (i.e. `169.254.169.1`).
+This route looks like: `169.254.169.1/32 dev breth0 mtu 1400 src <node IP address>`
+
+### Node masquerade IPs / routes
+In order to enable the host to services, we set the host masquerade IP (169.254.169.2) on the br-ext bridge.
+Since the br-ex bridge already has the node masquerade IP address, it  is no longer required to add the masquerade
```

Comment:
nit: double spacing

`no longer required` is a temporal reference to some other situation in the past, but the reader is not necessarily aware of this situation. Is this referencing another documentation? Should we add this reference? Otherwise, can this be described in absolute terms?

Reply:
Done. Is it better now ?